MapboxVision/src/main/java/com/mapbox/vision/video/videosource/camera/Camera2VideoSourceImpl.kt

```diff
@@ -178,6 +182,17 @@ class Camera2VideoSourceImpl(
         }
     }
 
+    private fun repeatConfigureAttempt() {
+        if (configureAttemptNumber <= CREATE_SESSION_MAX_ATTEMPTS) {
+            backgroundThreadHandler.handler.postDelayed({createCaptureSession()}, TimeUnit.SECONDS.toMillis(configureAttemptNumber * 2L))  // delay 2 - 4 - 6 seconds
+        } else {
+            configureAttemptNumber = 0
+            cameraDevice?.close()
```

Comment:
I meant lines 
```
cameraDevice?.close()
cameraDevice = null
cameraOpenCloseLock.release()
```
that are used in two other places, too

Reply:
fixed